addon/components/fixtable-grid.js

```diff
@@ -159,15 +163,24 @@ export default Ember.Component.extend({
       var filters = this.get('filterToApply') || {};
       var sortedFilteredContent = sortedContent.filter(function(row) {
         return Object.keys(filters).every(function(columnKey) {
-          if (!filters[columnKey]) { return true; } // no filter
-          var cellData = (((row.get) ? row.get(columnKey) : row[columnKey]) || '').toLowerCase();
-          var filterValue = filters[columnKey].toLowerCase();
+          var filterDef = columnsByKey[columnKey].filter;
+          var filterFunction = filterDef.function;
```

Comment:
This is definitely an edge case, but I've previously seen cases where using a JavaScript keyword (like `function`) as an object key has caused issues during minification. Just to be safe, could we rename this property in the column/filter configuration to `filterFunction` or something like that?